Framework/DataHandling/src/SaveToSNSHistogramNexus.cpp

```diff
@@ -232,9 +232,9 @@ int SaveToSNSHistogramNexus::WriteOutDataOrErrors(
   if (!doErrors) {
     // Add an attribute called "errors" with value = the name of the data_errors
     // field.
-    NXname attrName = "errors";
+    NXname NX_class = "errors";
```

Comment:
Local search and replace gone global? Causing latest round of build failures.

Reply:
Likely, dont know when it decided to go global